src/scss/components/optional/_order-card.scss

```diff
@@ -39,6 +39,14 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
             text-decoration: none;
             color: $grey--mid;
         }
+
+        .c-yourFavourites {
```

Comment:
Have you got some example markup of how this is used?

If this is a child element of the `c-orderCard` component, it should have a classname of `c-orderCard-something` to indicate that it's got a relationship between the parent class and the element being styled.